crates/bevy_ecs/src/system/system_param.rs

```diff
@@ -581,7 +632,16 @@ impl<'a, T: Component> SystemParamFetch<'a> for RemovedComponentsState<T> {
     }
 }
 
-/// Shared borrow of a NonSend resource
+/// Shared borrow of a non-[`Send`] resource.
+///
+/// Only `Send` resources may be accessed with the [`Res`] [`SystemParam`]. In case that the
+/// resource does not implement `Send`, this `SystemParam` wrapper can be used. This will instruct
+/// the scheduler to instead run the system on the proper thread so that it doesn't send the resource
```

Comment:
I dont like the usage of `proper` here. Perhaps use `main` here instead.

Reply:
Agree, that is clearer. Will get to that later.